keyboards/westfoxtrot/prophet/config.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x21FF
```

Comment:
This vendor ID is already in use:
https://www.the-sz.com/products/usbid/index.php?v=0x21FF